audio-worklet/src/bitcrusher-demo.js

```diff
@@ -124,13 +150,13 @@ class BitcrusherDemo {
    */
   switchBitcrusherNode() {
     // The function is called after the checkbox switches state.
-    const changeToScriptProcessor = !this.useAudioWorkletCheckBox_.checked;
-    if (changeToScriptProcessor) {
-      this.bitcrusherScriptProcessor_.output.connect(this.masterGain_);
-      this.bitcrusherAudioWorklet_.disconnect();
-    } else {
+    const changeToAudioWorklet = !this.useScriptProcessorCheckBox_.checked;
+    if (changeToAudioWorklet) {
       this.bitcrusherAudioWorklet_.connect(this.masterGain_);
       this.bitcrusherScriptProcessor_.output.disconnect();
+    } else {
+      this.bitcrusherScriptProcessor_.output.connect(this.masterGain_);
+      this.bitcrusherAudioWorklet_.disconnect();
```

Comment:
I had thought you meant we would be switching between the node and the masterGain, but it sounds like you are saying that the script processor and audio worklet each have dedicated gain nodes and we switch between their respective gain nodes and the masterGain? For the script processor that would mean three gain nodes separating it from the destination since its output node is a gain node.